koku/api/tags/ocp/queries.py

```diff
@@ -69,3 +83,31 @@ def __init__(self, parameters):
             parameters.set_filter(**{"enabled": True})
         # super() needs to be called after _mapper is set
         super().__init__(parameters)
+
+    def _get_key_filter(self):
```

Comment:
So we don't have a lot of repetitive code, let's move this function into tags/queries.py and save the QueryFilter params somewhere. Maybe a class attribute? e.g.
```
KEY_FILTERS = [
    {"field": "ocpusagepodlabelsummary__key"},
    {"field": "ocpstoragevolumelabelsummary__key"},
]
```
Then the main function can just iterate thru this list to add the appropriate filters:
```
        filters = QueryFilterCollection()
        if self.parameters.get_filter("value"):
            for something in self.KEY_FILTERS:
                filters.add(
                    QueryFilter(
                        field=something.get("field"),
                        operation="exact",
                        parameter=self.key,
                        composition_key="tag_filter",
                    )
                )
        else:
            ...
```
With something like this in place, we can remove `_get_key_filter` from all of the individual queries files.

Reply:
Good idea, changed to this functionality.